docs/http-api.md

```diff
@@ -74,13 +98,19 @@ POST /render
 
 * header `X-Org-Id` required
 * maxDataPoints: int (default: 800)
-* target: mandatory. one or more metric names or patterns, like graphite.  
+* target: mandatory. one or more metric names or patterns, like graphite.
   note: **no graphite functions are currently supported** except that
   you can use `consolidateBy(id, '<fn>')` or `consolidateBy(id, "<fn>")` where fn is one of `avg`, `average`, `min`, `max`, `sum`. see
   [Consolidation](https://github.com/raintank/metrictank/blob/master/docs/consolidation.md)
 * from: see [timespec format](#tspec) (default: 24 ago) (exclusive)
 * to/until : see [timespec format](#tspec)(default: now) (inclusive)
 
+#### Example
+
+```bash
+curl -H "X-Org-Id: 12345" "http://localhost:6060/metrics/render?target=statsd.fakesite.counters.session_start.*.count&from=3h&to=2h"
+```
+
 ## Cluster status
```

Comment:
While you're at it, could you call this `Get cluster status` and then the post below should get the title `Set cluster status`, it's a bit weird to have both in the same section